src/migrations/V40__add_meaningful_data.sql

```diff
@@ -0,0 +1,384 @@
+TRUNCATE TABLE public.kpi_definition, public.circle_kpi_definition, public.kpi_values_history, public.target, public.target_history RESTART IDENTITY;
+
+insert into public.kpi_definition (
+    kpi_name,
+    periodicity,
+    description,
+    unit,
+    value_min,
+    value_max,
+    active,
+    cumulative,
+    formula,
+    created_by,
+    is_approved
+) values 
+('Number of counseling sessions', 'weekly', 'Total counseling sessions held in a period.', 'numeric', 0, NULL, 'TRUE', 'FALSE', 'aggregate', '6ab40640-8ff6-4363-a2f5-5691df9b8821', 'TRUE'),
```

Comment:
yikes - user ids again ;). Maybe we can just run this script manually and keep it in a separate directory? Otherwise setting up the project from scratch will be broken again

Reply:
you're right sorry about that, I ran it from another branch already though to test, but I deleted it from the flyway_schema_history, should that be enough or do I have to roll back the changes somewhere?
If it's ok like that, I'll just run it locally again and save it to a data folder instead of migrations

Reply:
No worries, there shouldn't be a problem with our current "production" database - I would just move the file (and maybe rename it as well) so that Flyway doesn't pick it up.

Reply:
yes perfect, will do, thanks! :)

Reply:
ok, I added a new folder data in scr and put the sql file there and deleted it in the migrations folder